tslint.json

```diff
@@ -1,5 +1,6 @@
 {
     "rules": {
+        "align": [true, "parameters", "arguments", "statements"],
```

Comment:
Let's format this similarly to the other configuration options.  See `typedef` for example.